crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -385,6 +385,7 @@ where
         ));
         if key.mesh_key.contains(MeshPipelineKey::DEPTH_CLAMP_ORTHO) {
             shader_defs.push("DEPTH_CLAMP_ORTHO".into());
+            shader_defs.push("PREPASS_FRAGMENT".into());
```

Comment:
Can you add a `// PERF:` comment here explaining that this will force the prepass fragment shader to run for things like directional lights, and link to this PR?

This seems like the kind of thing that would be very easy to forget.

Reply:
Resolved!

Reply:
Thanks @cart!